contrib/charts/dragonfly/Chart.yaml

```diff
@@ -21,7 +21,7 @@ version: v1.0.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v1.0.0"
+appVersion: "v1.0.1"
```

Comment:
I don't think this is necessary, as we will update this version with a release candence containing all changes 👀